operator/v1/types_ingress.go

```diff
@@ -536,6 +536,43 @@ type HostNetworkStrategy struct {
 	// +kubebuilder:validation:Optional
 	// +optional
 	Protocol IngressControllerProtocol `json:"protocol,omitempty"`
+
+	// HTTPPort is the port on the host which should be used to listen for
```

Comment:
```suggestion
	// httpPort is the port on the host which should be used to listen for
```
https://github.com/openshift/enhancements/blob/master/CONVENTIONS.md#use-json-field-names-in-godoc (soon to be https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md#use-json-field-names-in-godoc).